lang/fpc/Portfile

```diff
@@ -233,7 +252,20 @@ if {${subport} eq "${name}"} {
 
     installs_libs       yes
 
-    extract.only-append ${pp}
+    if {${os.platform} eq "darwin" && ${configure.build_arch} in [list ppc ppc64]} {
+        post-extract {
+            xinstall -d /tmp/boot
+            system "hdiutil attach ${distpath}/${pp} -private -nobrowse -mountpoint /tmp/boot"
+            copy /tmp/boot/${name}-${boot_v}.powerpc-macosx.pkg ${workpath}/pkg
+            system "hdiutil detach /tmp/boot -force"
+            delete -force /tmp/boot
+            system -W ${workpath} "gunzip -d pkg/Contents/Archive.pax.gz"
+            system -W ${workpath}/pkg "pax -r < Contents/Archive.pax"
+            copy ${workpath}/pkg/usr/local/lib/${name}/${boot_v}/${bootstrapCompiler} ${workpath}
+        }
```

Comment:
I don't love the use of a path outside the build directory here. Is there a good reason not to put `boot` or whatever you call it inside the build directory itself?

Reply:
@pmetzger I believe this chunk is based on some existing example of another port. I did not try to modify it beyond what was required to make it serve the purpose.

Could you suggest a specific change? I will try it and commit.

Reply:
It would have been wrong to do this in another port as well. Unpacking into /tmp/ means that an existing directory of that name could be there, some other process could trample on the build, etc. It's preferable to do something like `${workpath}/bootapp/` or something. It doesn't matter much what, just not `/tmp/boot` or the like.

Reply:
Ok, let me fix that.